node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/yum/YumCommand.java

```diff
@@ -47,11 +48,18 @@ public T enableRepo(String... repo) {
         return getThis();
     }
 
+    /** Enables the given repos for this command */
+    public T disableRepo(String... repo) {
```

Comment:
Consider special-casing an empty list, to disable all (add List.of("*")).

Reply:
nice!

Reply:
The `"*"` triggers escaping , which leads to `yum install --assumeyes "--disablerepo=*" --enablerepo...`, I guess that it will cause the command to fail?

Reply:
Tested with escaped, and it works
```
sudo yum install "--disablerepo=*" --enablerepo
...
Complete!
```